packages/common-ui-web/src/components/TakePictureButton/TakePictureButton.css

```diff
@@ -0,0 +1,45 @@
+:root {
+  --take-picture-btn-border: 3px;
+  --take-picture-btn-color: #f3f3f3;
+  --take-picture-btn-color-hover: #e3e3e3;
+  --take-picture-btn-color-active: #cbcbcb;
+  --take-picture-btn-color-disabled: #b6b6b6be;
+}
+
+.take-picture-btn {
+  border: var(--take-picture-btn-border) solid var(--take-picture-btn-color);
+  border-radius: 50%;
+  display: inline-block;
+  margin-right: 20px;
```

Comment:
remove this margin : it is the job of the component that USES this button to decide if it wants a margin or not